examples/parametric/plot_delta_lambda.py

```diff
@@ -0,0 +1,113 @@
+"""
+Plot delta_lambda for a grid.
+===============================
+
+This script demonstrates how to generate delta_lambda from a provided grid. 
+It includes the following steps:
+- Builds a parametric galaxy using make_sfzh.
+- Retrieves delta_lambda for the galaxy using the grid.
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import synthesizer
+
+import importlib
+
+package_name = "synthesizer"
+
+from synthesizer.filters import FilterCollection
+from synthesizer.grid import Grid
+from synthesizer.parametric import SFH, ZDist, Stars
+from synthesizer.parametric.galaxy import Galaxy
+from unyt import Myr
+
+if __name__ == "__main__":
+    # Get the location of this script, __file__ is the absolute path of this
+    # script, however we just want the directory
+    # script_path = os.path.abspath(os.path.dirname(__file__))
+
+    # Define the grid
+    grid_name = "test_grid"
+    grid_dir = "../../tests/test_grid/"
+    grid = Grid(grid_name, grid_dir=grid_dir)
+
+       # define filters
+    filter_codes = [
+        f"JWST/NIRCam.{f}"
+        for f in ["F090W", "F115W", "F150W", "F200W", "F277W", "F356W", "F444W"]
+    ]  # define a list of filter codes
+    filter_codes += [f"JWST/MIRI.{f}" for f in ["F770W"]]
+    fc = FilterCollection(filter_codes, new_lam=grid.lam)
```

Comment:
this filter collection doesn't appear to actually be used in this example?

Reply:
Thanks for pointing this out. The filter collection has been removed